### DISCUSSION TITLE
Welcome to join our interest group if you are interested to improve CSGHub!

### DISCUSSION THREAD

**Discussion by Rader** [OpenCSGs/csghub#604] [Announcements]
Dear community users,

Hello everyone! 🌟

Welcome to join our WeChat interest group, a platform specially designed for friends who are passionate about developing the CSGHub source code. Here, we'll explore the charm of git development together, delve into cutting-edge technologies in large model applications, and share and discuss exciting experiences in Golang development. We also warmly welcome friends interested in cloud-native technologies, Docker, and Kubernetes.

In this group, you'll not only be able to discuss and share best practices but also have the opportunity to participate in our regularly held open-source meetings and online exchange activities. Offline meetups will be coming soon, providing more opportunities for face-to-face sharing and learning.

If you're interested in joining, please leave your WeChat ID in the comments along with your areas of interest. Whether you are a seasoned developer or a newcomer, we warmly welcome your participation. We look forward to your joining, as we explore, learn, and grow together!

Let's work together to contribute our wisdom and creativity to the future of CSGHub! 🎉

Welcome aboard! ✨

The CSGHub Team

**Comment by youngbeom-shin** [comment]
申请加群🙋   微信号：Youngbeom-Shin

**Reply by Rader** [reply]
欢迎！
请问，您对哪个方向比较感兴趣？

**Reply by youngbeom-shin** [reply]
想 `深入了解大模型应用的前沿技术` ，感谢邀请！
